public/views/address.html

```diff
@@ -1,9 +1,9 @@
 <section data-ng-controller="AddressController">
+  <link rel=StyleSheet href="../src/css/common.css" type="text/css" media="screen">
```

Comment:
`common.css` is minified into `main.min.css` during `grunt compile`, so we don't need to include it again here.